docs/kratos/self-service/flows/account-recovery-password-reset.mdx

```diff
@@ -0,0 +1,237 @@
+---
+id: account-recovery-password-reset
+title: Account recovery and password reset
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl"
+import Mermaid from "@site/src/theme/Mermaid"
+import ApiWarning from "@site/src/theme/ApiWarning"
+import SelfServiceBrowserFlow from "@site/src/theme/SelfServiceBrowserFlow"
+import SelfServiceSpaFlow from "@site/src/theme/SelfServiceSpaFlow"
+import SelfServiceApiFlow from "@site/src/theme/SelfServiceApiFlow"
+
+import CodeTabs from "@site/src/theme/Code/CodeTabs"
+import {
+  getFlowMethodLinkWithErrors,
+  getFlowMethodLinkSuccess,
+  getFlowMethodLinkInvalidChallenge,
+  getFlowMethodLinkChallengeDone,
+} from "./code/recovery"
+import RenderFlow from "@site/src/theme/Code/RenderFlow"
+
+import DomainWarning from "@site/src/theme/DomainWarning"
+import AjaxWarning from "@site/src/theme/AjaxWarning"
+import Tabs from "@theme/Tabs"
+import TabItem from "@theme/TabItem"
+
+Account Recovery allows registered users to regain access to their account if they lost their password or multifactor
+authentication (MFA) provider.
+
+If a user loses the credentials to their account, they need to prove ownership of the account to recover access. This is typically
+done via a recovery address such as an email address or phone number. The user can prove their identity by accessing a link or
+one-time code sent to a recovery address.
+
+Ory Identities (Ory Kratos) supports two recovery methods:
+
+- Sending a one-time password (**preferred**)
+- Sending a "magic link"
+
+## Configuring self-service account recovery
+
+<Tabs
+  defaultValue="oc"
+  values={[
+    {label: 'The Ory Network', value: 'oc'},
+    {label: 'Self-hosted Ory Kratos config', value: 'oss'}
+  ]}>
+  <TabItem value="oc">
```

Comment:
1. Please wrap all MDX elements (such as <Tabs>) in ```mdx-code-block
2. The way you're doing the tab names is a bit too complicated I think, you can do it like this instead: 

```
   <Tabs groupId="react-or-preact">
     <TabItem value="react" label="React" default>
```

`groupId` allows you to set up the tabs so that all tabs are in sync
`default` allows you to decide which item is shown by default in the given tab group

Reply:
https://github.com/ory/docs/blame/master/docs/getting-started/custom-ui-preact-ory-elements.mdx#L42

Reply:
Great point!! Thanks! This is way better.